### DISCUSSION TITLE
Broken C++ API example link in doc

### DISCUSSION THREAD

**Discussion by Keezuu** [apache/incubator-mxnet#19566] [Q&A] [answered]
Hi all,
I want to write simple MXNet code in C++ API to run a model pretrained and exported in Python MXNet. Unfortunately the only doc I could find about that is this one https://mxnet.apache.org/versions/1.7.0/api/cpp/docs/tutorials/cpp_inference

The tutorial refers to example at this link https://github.com/apache/incubator-mxnet/tree/master/cpp-package/example/inference unfortunately, this link is broken and no longer leads to example code.
I've tried to find equivalent example in the repo but couldn't find one.

Is there any code example for C++ API inference with model loading (with params/symbols saved in Python's MXNet) in the repo or in docs?

**Comment by szha** [comment, marked as answer]
The correct link should be https://github.com/apache/incubator-mxnet/tree/v1.x/cpp-package/example/inference

**Reply by Keezuu** [reply]
Thanks alot!
